Code/Tools/SceneAPI/SceneUI/CommonWidgets/SceneSettingsCard.h

```diff
@@ -65,6 +65,8 @@ class SceneSettingsCardHeader : public AzQtComponents::CardHeader
 
     QPushButton* m_closeButton = nullptr;
     QSvgWidget* m_busySpinner = nullptr;
+    
+    friend class SceneSettingsCard;
```

Comment:
what is the justification for a friend class here? maybe the relationship with SceneSettingsCardHeader is wrong?

Reply:
I'm setting the tooltip on the busy spinner from the SceneSettingsCard. I could make a passthrough, but these two classes were so intertwined I figured it was fine using the friend instead. This also matches the pattern in CardHeader.h and Card.h, CardHeader is friends with Card.